tests/test_parser.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+from bankline_parser.data_services import parse_file, parse
+from bankline_parser.data_services.exceptions import ParseError
+
+
+class DataServicesParserTestCase(unittest.TestCase):
```

Comment:
Those kind of integrity tests are carried out in the validation methods (which are tested by ```assertTrue(output.is_valid())```).